src/App.jsx

```diff
@@ -1,12 +1,11 @@
-import React from 'react';
```

Comment:
from react .... not sure 16? on... we don't need to import explicitely React itself and we can just import the functions we need.